app/src/main/res/layout/item_media_folder.xml

```diff
@@ -30,11 +30,28 @@
         android:layout_marginStart="@dimen/marginStandard"
         android:ellipsize="end"
         app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintBottom_toTopOf="@id/mediaFolderPath"
         app:layout_constraintEnd_toStartOf="@id/mediaFolderSwitch"
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toTopOf="parent"
         tools:text="Camera" />
 
+    <TextView
+        android:id="@+id/mediaFolderPath"
+        style="@style/TextAppearance.Material3.BodySmall"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="@dimen/marginStandard"
+        android:ellipsize="end"
```

Comment:
Is it better to ellipsize the end of the path or the middle ? 
I think I have seen  file explorer ellipsizing the middlebut i'm not sure

Reply:
Middle is probably better for a file path.